spec/models/comment_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  context "relationships" do
+    it { should belong_to(:answers) }
```

Comment:
The assertions on lines five and six will fail with how the polymorphic association is built for comments.  It will belong_to 'commentable'.  I don't know of a way to assert that a commentable can be a question or an answer, but it seems like it could be inferred from model tests for answers and questions.